llvm/lib/Target/AIE/AIE2Subtarget.cpp

```diff
@@ -46,6 +46,7 @@ AIE2Subtarget::AIE2Subtarget(const Triple &TT, StringRef CPU, StringRef TuneCPU,
                              StringRef FS, StringRef ABIName,
                              const TargetMachine &TM)
     : AIE2GenSubtargetInfo(TT, CPU, TuneCPU, FS), AIEBaseSubtarget(TT),
+      AddrSpaceInfo(),
```

Comment:
I don't think we need this line, `AddrSpaceInfo` will already be default constructed.